Hippo.Web/ClientApp/src/Shared/RejectRequest.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+
+import { Account } from "../types";
+import { authenticatedFetch } from "../util/api";
+import { usePromiseNotification } from "../util/Notifications";
+import { useConfirmationDialog } from "./ConfirmationDialog";
+import { notEmptyOrFalsey } from "../util/ValueChecks";
+
+interface Props {
+  account: Account;
+  removeAccount: (account: Account) => void;
+  updateUrl: string;
+}
+
+export const RejectRequest = (props: Props) => {
+  //const history = useHistory();
+  const [reason, setReason] = useState("");
+
+  const [notification, setNotification] = usePromiseNotification();
+
+  const [getConfirmation] = useConfirmationDialog<string>(
+    {
+      title: "Reject Request",
+      message: (setReturn) => (
+        <div className="form-group">
+          <label htmlFor="fieldName">Reason</label>
+          <textarea
+            className="form-control"
+            id="fieldName"
+            rows={3}
+            required
+            value={reason}
+            onChange={(e) => {
+              setReason(e.target.value);
+              setReturn(e.target.value);
+            }}
```

Comment:
Should we do something here to enforce the 1500 char limit? Or should we wait until we have yup validation in place?

Reply:
That was me just pulling a number out of my hat... Email would still get the full text